scripts/generate_rest.sh

```diff
@@ -2,7 +2,6 @@
 
 command -v swagger >/dev/null 2>&1 || {
   echo >&2 "Command 'swagger' not installed. See: https://github.com/go-swagger/go-swagger for installation"
-  exit 1
```

Comment:
?

Reply:
commented out b/c it wasn't detecting a swagger alias. I meant to put it back, fixing.